Resources/Prototypes/Recipes/Reactions/fun.yml

```diff
@@ -93,7 +93,7 @@
     Licoxide: 1
 
 - type: reaction
-  id: PlasticSheet # It's here because "haha look you can get smooth beautiful sheets of plastic from small cylindric beaker" (jokes aside: i dont know where should i put it)
+  id: PlasticSheet # plastic steel and glass should have their own .yml file named "materials.yml". also note how all material reactions total to 10 reactants to follow balance
```

Comment:
Why don't you do that in this PR? Unless I'm missing something (Easily possible!) you should just be able to create that file in this directory and add the three reactions to it!

Reply:
I could do that as a last commit for organization. The problem is this whole PR is currently dividing the contributors on whether chemistry should be an alternative (but expensive) source of materials should cargo not be available. One one side Botany can currently make Steel, on the other side it is detrimental to both salvage and cargo balance as it undermines their purpose.